src/main/java/edu/iastate/ece/sd/sdmay2126/application/FBAParameters.java

```diff
@@ -16,7 +20,7 @@
     private float expressionThreshold;
     private float expressionUncertainty;
     // TODO geneKnockouts
-    // TODO reactionKnockouts
+    private LinkedList<String> reactionKnockouts;
```

Comment:
Do we want to add these new parameters to the constructor as we introduce them?

CC @DaultonLeach, did you add the other primitives to the constructor?